<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProbeTestDialog</class>
 <widget class="QDialog" name="ProbeTestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>531</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>名称</string>
   </property>
   <widget class="QLineEdit" name="batchName_lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>531</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>阻值</string>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>251</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>134</width>
      <height>14</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>测试次数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="testtimes_label">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>基片编号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="baseno_label">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>440</y>
      <width>328</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="sub10_pushButton">
       <property name="text">
        <string>-10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sub1_pushButton">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add1_pushButton">
       <property name="text">
        <string>+1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add10_pushButton">
       <property name="text">
        <string>+10</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>211</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="lRoptium_checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>低阻优化(&lt;10Ω)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Bias_checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>补偿测量</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Plate_comboBox">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <item>
     <property name="text">
      <string>T4(&lt;10MΩ)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>T5(&gt;10MΩ)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>测试平台</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>610</y>
     <width>245</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="Print_pushButton">
      <property name="text">
       <string>测试结果</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Pmeasure_pushButton">
      <property name="text">
       <string>测量</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Exit_pushButton">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
